modules/calib/src/calibinit.cpp

```diff
@@ -548,11 +539,7 @@ bool findChessboardCorners(InputArray image_, Size pattern_size,
 
         detector.reset();
 
-#ifdef USE_CV_FINDCONTOURS
         Mat binarized_img = thresh_img_new;
-#else
-        Mat binarized_img = thresh_img_new.clone(); // make clone because cvFindContours modifies the source image
-#endif
         detector.generateQuads(binarized_img, flags);
```

Comment:
maybe deleting `Mat binarized_img = thresh_img_new;` and `detector.generateQuads(thresh_img_new, flags);` will be better